pkg/option/config.go

```diff
@@ -3164,8 +3165,25 @@ func sanitizeIntParam(paramName string, paramDefault int) int {
 	return intParam
 }
 
+// validateConfigmap checks whether the flag exists and validate the value of flag
+func validateConfigmap(cmd *cobra.Command, m map[string]interface{}) {
+	// validate the config-map
+	for key, value := range m {
+		if val := fmt.Sprintf("%v", value); val != "" {
+			flags := cmd.Flags()
+			// check whether the flag exists
+			if flag := flags.Lookup(key); flag != nil {
+				// validate the value of flag
+				if err := flag.Value.Set(val); err != nil {
+					log.WithError(err).Fatal("Incorrect config-map flag " + key)
```

Comment:
nit: I'd change this function to return an error and move the `log.Fatal()` to `InitConfig()` to make this more explicit

Reply:
Thanks Gilberto

Done